### ISSUE TITLE
Support for file:///path/to/somefile

### ISSUE THREAD

**Issue by reikred** [minbrowser/min#1594]
The title says it all. I try typing in file:///path/to/somefile and it does not work. Should it not? Am I doing something wrong.

Bonus question: Command line "min file:///path/to/somefile " ought to work as well.

**Comment by PalmerAL** [comment]
This should work already; can you be more specific about what path you're typing? And what happens when you enter that path?

**Comment by reikred** [comment]
I just checked again, and typing in file:///home/myuser does not work. Firefox has noo problems with it. The error messagge is

An error occurred (-6)
￼￼Try again

Version: 1.14.1-1

-----------------------------------------------------------------------------------------------------

Maybe unrelated, maybe not:  I also see an error message in the terminal:

Error: ENOENT: no such file or directory, open '/home/myuser/.config/Min/settings.json'

This error message goes away if I do

touch /home/myuser/.config/Min/settings.json

**Comment by reikred** [comment]
Yes, I see now. Opening an individual html file does work, as in 

min file://path/to/somefile.html
min file://`pwd`/somefile.html

"Would be nice" if directories worked as well, but being minimalist is also a nice objective ;-). Thanks.
